### PR TITLE
hub: make the footer slightly bigger

### PR DESCRIPTION
**Pull request by siteshwar**
... to make it easier to read.

### CODE REVIEW

**Comment by siteshwar** [comment]
Marking it as draft, as I could not test it due to following exception:

```
+ podman-compose -p osh up --build -d db osh-hub osh-worker osh-client
Traceback (most recent call last):
  File "/usr/bin/podman-compose", line 33, in <module>
    sys.exit(load_entry_point('podman-compose==1.1.0', 'console_scripts', 'podman-compose')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 3308, in main
    asyncio.run(async_main())
  File "/usr/lib64/python3.12/asyncio/runners.py", line 194, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib64/python3.12/asyncio/base_events.py", line 687, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 3304, in async_main
    await podman_compose.run()
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 1634, in run
    retcode = await cmd(self, args)
              ^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 2310, in compose_up
    excluded = get_excluded(compose, args)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/site-packages/podman_compose.py", line 2301, in get_excluded
    excluded -= compose.services[service]["_deps"]
                ~~~~~~~~~~~~~~~~^^^^^^^^^
KeyError: 'osh-client'
```

**Comment by lzaoral** [comment]
@siteshwar You can test and finish this draft commit for the CI failure: https://github.com/openscanhub/openscanhub/pull/288/commits/c4490c02acc4de0890e4bbd3c7a7218040db62ff

I won't have time to look at it in the near future.

File: osh/hub/static-assets/css/redhat.css [hunk 1]
```diff
@@ -793,3 +793,14 @@ div#defects div.events_list div.code {
 }
 
 /**/
+
+
+/**/
```
Comment (kdudka) [inline osh/hub/static-assets/css/redhat.css:798]: Why do you insert the empty comments and the doubled empty lines?

Comment (siteshwar) [reply osh/hub/static-assets/css/redhat.css]: Moved the changes to another block.

**Review by kdudka — APPROVED** [review]

**Pull request merged**